ansible/roles/benchmark/tasks/install.yml

```diff
@@ -14,6 +14,16 @@
 - name: Install jq
   package: name=jq state=present
 
+- name: Install dig
+  package: name=bind-utils state=present
+
+- name: Download & install OpenShift client
```

Comment:
I don't think you are using `oc` anywhere.

Reply:
We need `oc` to login to the Openshift cluster before we execute the `kc-failover.sh` script.